src/language-css/utils/stringify-node.js

```diff
@@ -5,7 +5,7 @@ function stringifyNode(node) {
     const open = node.open?.value || "";
     const groups = node.groups
       .map((currentValue) => stringifyNode(currentValue))
-      .join(node.groups[0]?.type === "comma_group" ? "," : "");
+      .join(node.type === "paren_group" ? "," : "");
```

Comment:
Commas had disappeard if the first comma group is flattened.
Groups in paren group should always be separated by comma.